javascript_nodejs/14.nlp-with-dispatch/dialogs/qna/index.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const { QnAMaker } = require('botbuilder-ai');
+
+// this is the LUIS service type entry in the .bot file.
```

Comment:
Capitalize. "This" or (better) clean up the grammar...

Reply:
Fixed.